pkg/store/image/image.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"sync"
+
+	imagespec "github.com/opencontainers/image-spec/specs-go/v1"
+
+	"github.com/kubernetes-incubator/cri-containerd/pkg/store"
+)
+
+// Image contains all resources associated with the image. All fields
+// are **read-only**.
```

Comment:
Is that true? In `Add()`, you merge RepoTags and RepoDigest and write it back to the store.